ibis/backends/pyspark/tests/test_aggregation.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+import pytest
+from pytest import param
+
+pytest.importorskip("pyspark")
+
+
+@pytest.mark.parametrize(
```

Comment:
I put it in the PySpark-backend specific tests because we currently don't have any common test datasets that have nulls or NaNs. I could add a dataset like this then make this a generic test—would adding a new test dataset be any more involved than adding a new .csv to https://github.com/ibis-project/testing-data?

Reply:
Yeah, you'd have to add that new table in `ci/schema` for each of the `.sql` files in there.

Reply:
Can you leave this as is but only test one or two aggregate functions? I'll refactor this into the backend suite later.

Reply:
sure I can do that, thank you